src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-custom-tag.js

```diff
@@ -274,6 +278,8 @@
                 unsetAlignment: function() {
                     this.wrapper.data(DATA_ALIGNMENT_ATTR, false);
                     this.element.data(DATA_ALIGNMENT_ATTR, false);
+                    window.clearTimeout(this.unsetAlignmentFireEditorInteractionTimeout);
+                    this.unsetAlignmentFireEditorInteractionTimeout = window.setTimeout(this.fireEditorInteraction.bind(this,'aligmentRemoved'), 50);
```

Comment:
Those lines seem to be too long. Do you use Prettier?

Reply:
Nope, I will in the future, thanks!